fiftyone/core/collections.py

```diff
@@ -10601,7 +10750,15 @@ def _parse_aggregations(self, aggregations, allow_big=True):
         big_aggs = {}
         batch_aggs = {}
         facet_aggs = {}
+        stream = None
+
         for idx, aggregation in enumerate(aggregations):
+            # stream is True if all aggregations are lazy
```

Comment:
Suppose I do this:

```py
results = dataset.aggregate([fo.Values(..., _lazy=True), fo.Values(...)])
```

Since only one aggregation is "lazy", I'll get `stream=False` in this method, which means that the results of `fo.Values(..., _lazy=True)` will get parsed by the "non-lazy" implementation. But the lazy aggregation's `parse_result()` method will return a lambda (because it still thinks it will be used in "lazy" mode), not the values.

So the result will be non-sensical.